File: Dtos/CategoryDto.cs
namespace Productivity.Dtos;

public class CategoryDto
{
    public long? Id { get; set; }
    public string Description { get; set; }
    public string Color { get; set; }
    public bool IsNoteCategory { get; set; }
    public bool IsListCategory { get; set; }
    public string UserId { get; set; }
}

File: ClientApp/src/dtos/CategoryDto.js
export class CategoryDto {
    
    constructor(id, description, color, isNote, isList, userId) {
        this.id = id || null;
        this.description = description;
        this.color = color;
        this.isNoteCategory = isNote;
        this.isListCategory = isList;
        this.userId = userId;
        
    }
}